### ISSUE TITLE
Add (optional) indication to reports if pathers/pirates/warning systems

### ISSUE THREAD

**Issue by stormbringer951** [stormbringer951/CaptainsLog#48]
Check beacon

**Comment by jaghaimo** [comment]
Beacon for remnants works, but conditions for Pather / Pirates can change - if base despawns (or rather gets destroyed by the player) their presence will disappear.

**Comment by stormbringer951** [comment]
> Beacon for remnants works, but conditions for Pather / Pirates can change - if base despawns (or rather gets destroyed by the player) their presence will disappear.

Yeah, I would need to check the logic in those situations (it should follow same logic as pirate/pather base disappearing) since I would like to not leak information. I think those get updated if you're in comm range upon base destruction (would need to check with Nex AI pirate removal fleets etc)?

**Comment by jaghaimo** [comment]
It's going to be leaked if you update comm's ownership (pirates or pathers will claim it). Personally, I'd not add this at all, nor add comm relay ownership.
